public/vendor/prequel/mix-manifest.json

```diff
@@ -0,0 +1,4 @@
+{
+    "/app.js": "/app.js",
+    "/home/aether/Documents/Projects/HobbySoftware/laravel-sequel/packages/public/vendor/prequel/app.js": "/home/aether/Documents/Projects/HobbySoftware/laravel-sequel/packages/public/vendor/prequel/app.js"
```

Comment:
Is this line specific to your machine?

Reply:
Another reason these need to be `generated` and not stored in VCS :) 

@WindOfRussia

Reply:
ouf, that is a line specific to the package developer's machine.

Reply:
it is also a mix manifest that is never used by anyone than the package developer so not critical but definitely useless to keep in vcs.